ingest/scripts/split-gisaid-ndjson-by-segment

```diff
@@ -159,3 +166,10 @@ if __name__ == '__main__':
         Path(args.output_metadata).touch()
     else:
         write_records_to_tsv(chain([first_record], modified_records), args.output_metadata)
+
+    # Summary statistics (to STDERR)
+    print_err(f"Num records: {sum(STATS.values()):,}")
+    print_err("Num segments per record:")
+    print_err(f"0 segments: {STATS[0]:,} (records not written to metadata)")
+    for i in range(1, 8+1):
+        print_err(f"{i} segments: {STATS[i]:,}")
```

Comment:
from https://github.com/nextstrain/seasonal-flu/pull/274#pullrequestreview-3565191415:

> Idea: expand the stats in split-gisaid-ndjson-by-segment to also report counts of sequences (for each segment)

I'm not 100% sure what you mean here, so I'll come back to this later.